src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -192,11 +192,11 @@ export class DictionaryModal extends React.Component {
                       value={data.public_access}
                     >
                       <option value="" />
-                      {(isEditingDictionary && dictionary.public_access === 'View') &&
-                      <option value="View" selected>Private </option>
+                      {(isEditingDictionary && dictionary.public_access === 'None') &&
+                      <option value="None" selected>Private </option>
                       }
-                      <option value="Edit">Public</option>
-                      <option value="View">Private</option>
+                      <option value="View">View</option>
+                      <option value="None">None</option>
```

Comment:
What does `none` mean to the end user? I think using Private and Public sends a clearer message but also put into consideration the review from @alexmochu.

Reply:
Seek clarity from @alexmochu on the naming here.

Reply:
@dondrzzy @kmunene This change was requested by the Product Owner, he wanted the 'Public' and 'Private' options to be changed to 'View' and 'None'.